### PR TITLE
README, Index Page and User Stories and Use Cases in DG

### LINKED ISSUE #236: Update the user stories and use case for DG

### LINKED ISSUE #239: Update README and index page

### PR DESCRIPTION
**Pull request by frisciliasultan**
- Update README description and update UI screenshot
- Update index page description and update UI screenshot
- Update user stories
- Add use cases for client in DG

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,16 +3,19 @@
...
-* Gender
-* Age
-* Custom Information (e.g. food preferences)
+* Contact Number
+* Address
+* Email address
+* Insurance plan
+* Custom tags (e.g. food preferences, premium status, etc)
+* Profile Picture
 * Appointments you have with them
```
Comment (Tomashiwa) [inline README.md]: Maybe change `Appointments` to `Meetings` ?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -416,17 +419,25 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+| `* * *`  | accommodating insurance agent              | reschedule my meeting to another date                   | change my meeting to another date if client cannot make it on the specified date|
+| `* * *`  | insurance agent                            | delete a meeting                                        | cancel meetings that I no longer need                                  |
+| `* * *`  | busy insurance agent                       | find a meeting by client                                | locate details of meetings without having to go through the entire list|                                 |
+| `* * *`  | time-constrained insurance agent           | find a meeting by date                                  | list the meetings I have on a specific date                            |                                 |
+| `* * `   | active insurance agent                     | track my clients’ insurance expiry                      | remind my clients to renew their contract                              |
```
Comment (Tomashiwa) [inline docs/DeveloperGuide.md:446]: Just to confirm, Insurance got an expiry date? I can't find it in the InsurancePlan class.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -416,17 +419,25 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+| `* *`    | insurance agent with many contacts         | save my client’s likes/dislikes                         | interact with them more effectively and increase the chance of insurance sale |                          |
```
Comment (Tomashiwa) [inline docs/DeveloperGuide.md:447]: Not sure if the testers will mistake saving of likes/dislikes as a feature we were going to do

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -416,17 +419,25 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+| `*`      | user with many clients in the location book| sort clients by name                                    | locate a client easily                                                 |
+| `*`      | user                                       | hide private contact details                            | minimize chance of someone else seeing them by accident                |
```
Comment (Tomashiwa) [inline docs/DeveloperGuide.md:449]: Similar to the likes/dislikes, "hide private contact details" may be misunderstood as a potential feature

**Review by Tomashiwa — COMMENTED** [review]
Should be GTM once you address the user stories issues that I pointed out in the review.

File: README.md (hunk 1 above)
Comment (frisciliasultan) [reply README.md]: Nice catch, thanks

**Review by Tomashiwa — APPROVED** [review]
LGTM.

**Pull request merged**